assets/data/contributors.js

```diff
@@ -1,37 +1,44 @@
-[
+export const contributorsArray = [
+  {
+    name: 'Sam Craig',
+    linkedIn: 'https://www.linkedin.com/in/sam-craig-059834252/',
+    gitHub: 'https://github.com/Samcraig77',
+    avatar: '/assets/images/contributors/Sam-avatar.jpg',
+    alt: 'Sam looking at the camera, with an out of focus tree in the background.'
```

Comment:
Nice addition! I will add a comment on issue #26 to include alt description if sharing a picture.

Reply:
Sorry, I went into the mountains (literally) after I submitted haha. Bad to submit then dip I know, sorry! Would love to work on any issues found 😊